py4j-java/src/main/java/py4j/GatewayServer.java

```diff
@@ -754,7 +783,7 @@ protected void startSocket() throws Py4JNetworkException {
 	 * @param interfacesToImplement
 	 * @return
 	 */
-	public Object getPythonServerEntryPoint(Class[] interfacesToImplement) {
+	public Object getPythonServerEntryPoint(@SuppressWarnings("rawtypes") Class[] interfacesToImplement) {
```

Comment:
I was wondering about the method parameter annotation and not SupressWarnings in particular, but it seems that it is indeed supported (CircleCI will catch it anyway).